src/Packs.sol

```diff
@@ -782,17 +794,22 @@ contract Packs is
 
     /// @notice Handles receiving ERC721 tokens
     /// @dev Required for contract to receive ERC721 tokens via safeTransferFrom
-    function onERC721Received() external pure returns (bytes4) {
+    function onERC721Received(
```

Comment:
If we aren't holding ERC721 on the contract why do we need to send the success selector back? This would enable a `safeTransfer...` and then we would have to rescue the token.